packages-ts/gauntlet-terra-contracts/src/commands/abstract/index.ts

```diff
@@ -194,10 +211,16 @@ export default class AbstractCommand extends TerraCommand {
     const signer = this.wallet.key.accAddress // signer is the default loaded wallet
     const contractAddress = this.args[0]
     const input = this.params
-    const msg = new MsgExecuteContract(signer, contractAddress, input)
+    const msgs: MsgExecuteContract[] = []
```

Comment:
We shouldn't need to change the Abstract command. It is doing only one thing, one input, one message, one transaction. The smallest unit of logic of our framework. Changing it to handle multiple messages, actions, txs will complicate its usage